roles/docker/defaults/main.yml

```diff
@@ -29,3 +29,6 @@ r_crio_os_firewall_deny: []
 r_crio_os_firewall_allow:
 - service: crio
   port: 10010/tcp
+
+
+openshift_docker_is_node_or_master: "{{ True if inventory_hostname in (groups['oo_masters_to_config']|default([])) or inventory_hostname in (groups['oo_nodes_to_config']|default([])) else False | bool }}"
```

Comment:
Couldn't that be shortened (a little) to:

``{{ True if 'oo_masters_to_config' in group_names or 'oo_nodes_to_config' in group_names else False }}``

?

Reply:
This could probably be done.